src/daemon/daemon.cpp

```diff
@@ -99,7 +100,7 @@ mp::Query query_from(const mp::LaunchRequest* request, const std::string& name)
 }
 
 auto make_cloud_init_vendor_config(const mp::SSHKeyProvider& key_provider, const std::string& time_zone,
-                                   const std::string& username)
+                                   const std::string& username, const std::string& backend_version_string)
```

Comment:
This could be kept as a `QString`, since support was added to fmt. But I understand it looks cleaner to deal with only one string type here, so I am happy with it.

Reply:
Yeah we'd have to move them all to QString.